lib/build.api.js

```diff
@@ -213,26 +192,22 @@ class Build {
             fs.mkdirSync(destBase);
```

Comment:
In my PR I actually create the directory recursivly here and in the symlink section if doesn't exist.